stock_barcodes/wizard/stock_barcodes_read_picking.py

```diff
@@ -601,6 +601,10 @@ def _prepare_scan_log_values(self, log_detail=False):
 
     def remove_scanning_log(self, scanning_log):
         for log in scanning_log:
+            if log.picking_id.picking_type_code == "incoming":
+                rounding = log.product_id.uom_po_id.rounding
```

Comment:
An incoming picking can come from a return of an OUT, so PO UoM may not apply here.

Reply:
@sergio-teruel can you check this?